addons/event/static/src/views/event_registration_list_controller.js

```diff
@@ -0,0 +1,41 @@
+import { EventRegistrationSummaryDialog } from "@event/client_action/event_registration_summary_dialog";
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+import { listView } from "@web/views/list/list_view";
+import { ListController } from "@web/views/list/list_controller";
+
+export default class EventRegistrationListController extends ListController {
+
+    setup() {
+        super.setup();
+        this.dialog = useService("dialog");
+        this.orm = useService("orm");
+    }
+
+    async openRecord(record, mode) {
```

Comment:
is there a 'mode' argument on that method in the ListController?

Reply:
Aahh! missed that... it's for the kanban only. :grimacing: